extensions/tidal-controller/package.json

```diff
@@ -56,6 +56,80 @@
       "title": "Show Messages",
       "description": "Show pop up messages when an action is performed, such as toggling play/pause. Will also show a message if the action fails for some reason. No messages will be shown (even errors) if this is disabled.",
       "default": true
+    },
+    {
+      "name": "language",
+      "label": "Choose Language",
+      "type": "dropdown",
+      "required": true,
```

Comment:
Can we set this `false` and set English as default? This way the user will see the error message when trying to use the command and we won't change the flow for many users using it today